tests/testsuite/install.rs

```diff
@@ -1997,8 +2001,11 @@ fn install_global_cargo_config() {
 
     cargo_process("install bar")
         .with_status(101)
-        .with_stderr_contains("[INSTALLING] bar v0.0.1")
-        .with_stderr_contains("[..]--target nonexistent[..]")
+        .with_stderr_data(str![[r#"
+...
```

Comment:
`[INSTALLING] bar v0.0.1` is missing, though I think that is fine.

Reply:
I'll double-check this and re-add it.